OsmAnd/res/values/strings.xml

```diff
@@ -560,9 +560,9 @@
     <string name="show_passed">Show passed</string>
     <string name="hide_passed">Hide passed</string>
     <string name="remove_from_map_markers">Remove from map markers\'</string>
-    <string name="descendingly">descending</string>
-    <string name="ascendingly">ascending</string>
-    <string name="date_added">Date added</string>
+    <string name="ascendingly">A-Z</string>
```

Comment:
Not 100% sure these are always letters, but then again the point gets across that A= 0 if it is sometimes numbers